src/fmu/datamodels/common/access.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from pydantic import (
+    BaseModel,
+    Field,
+)
+
+from fmu.datamodels.fmu_results import enums
+from fmu.datamodels.fmu_results.fields import Asset, Ssdl
+
+
+class Access(BaseModel):
+    """
+    The ``access`` block contains information related to access control for
+    this data object.
+    """
+
+    asset: Asset
+    """A block containing information about the owner asset of these data.
+    See :class:`Asset`."""
+
+    classification: enums.Classification | None = Field(default=None)
+    """The access classification level. See :class:`enums.Classification`."""
+
+
+class SsdlAccess(Access):
+    """
+    The ``access`` block contains information related to access control for
+    this data object, with legacy SSDL settings.
+    """
+
+    ssdl: Ssdl
+    """A block containing information related to SSDL. See :class:`Ssdl`."""
```

Comment:
This wasn't super clear from the issue, but I think all models that are nested inside of these moved models should also be moved into `common/`. Ideally, stuff in `common/` never needs to import models or enums from outside of it. The relationship should be one way: stuff in common is only ever _imported_